tools/install_prereqs.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# Build MIGraphX prerequisites for docker container
+
+#install pip3
+sudo apt update && sudo apt install -y python3-pip
```

Comment:
Since this is for the dockerfile we can remove sudo, since it will already run the script as sudo.